library/swift/src/GRPCStreamEmitter.swift

```diff
@@ -43,7 +43,9 @@ public final class GRPCStreamEmitter: NSObject {
   }
 
   /// Close this connection.
-  public func close() {
-    self.underlyingEmitter.close()
+  public func close(_ data: Data) {
```

Comment:
we dont need to expose the data here, open to just having self.underlyingEmitter.closeWithData(Data()) below

Reply:
Just commented with that suggestion - I think we should avoid exposing the option to close with custom data